lib/http.js

```diff
@@ -79,23 +99,26 @@ function httpRequest(sdk, options) {
     });
 }
 
-function get(sdk, url, saveResponse) {
+function get(sdk, url, options) {
   url = util.isAbsoluteUrl(url) ? url : sdk.options.url + url;
-  return httpRequest(sdk, {
+  var getOptions = {
     url: url,
-    method: 'GET', 
-    dontSaveResponse: !saveResponse
-  });
+    method: 'GET'
+  };
+  util.extend(getOptions, options);
+  return httpRequest(sdk, getOptions);
 }
 
-function post(sdk, url, args, dontSaveResponse) {
+function post(sdk, url, args, options) {
   url = util.isAbsoluteUrl(url) ? url : sdk.options.url + url;
-  return httpRequest(sdk, {
+  var postOptions = {
     url: url,
     method: 'POST',
     args: args,
-    dontSaveResponse: dontSaveResponse
-  });
+    cacheState: true
```

Comment:
Ah, I think I see what's going on here - the "dontSaveResponse" from before was the idea of remembering the previous response in the authn flow. 

I think "cacheState" is a confusing variable name now that we have a cache that persists past the current auth flow.